### DISCUSSION TITLE
Welcome to pipupgrade Discussions!

### DISCUSSION THREAD

**Discussion by achillesrasquinha** [achillesrasquinha/pipupgrade#114] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xerxes24mg** [comment]
Hi,

With pipupgrade it is not possible "at the moment" to ugrade all python packages with python@3.12, is that right? Will it be one in the near future?

Thanks for your answer,
Michael
